<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>671</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>回测条件：</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>631</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="4">
      <widget class="QDateEdit" name="dt_end">
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2018</year>
         <month>1</month>
         <day>1</day>
        </datetime>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="btn_backtest">
       <property name="text">
        <string>开始回测</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QSpinBox" name="box_max_hold">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QLineEdit" name="edit_init_cash">
       <property name="text">
        <string>100000</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDateEdit" name="dt_start">
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2017</year>
         <month>1</month>
         <day>1</day>
        </datetime>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>起始资金：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>时间：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>——</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>基准：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <widget class="QComboBox" name="combo_benchmark"/>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>组合最大持仓数：</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
